MCMC/simulation.py

```diff
@@ -51,6 +53,8 @@ def __init__(
         self.energies = [copy.deepcopy(self.energy)]
         self.temperatures = []
         self._tps = []
+        
+        random.seed(seed=seed)
```

Comment:
Getting an error on this line.

Reply:
Thanks for pointing that out. I changed `seed=seed` to just `seed` and the error went away.